apis/python/src/tiledbsoma/_exception.py

```diff
@@ -10,7 +10,7 @@
 class SOMAError(Exception):
     """Base error type for SOMA-specific exceptions.
 
-    Lifecycle: maturing
+    Lifecycle: Maturing
```

Comment:
```suggestion
    Lifecycle: Maturing.
```

The other files have a '.' after the lifecycle.